package.json

```diff
@@ -30,17 +30,17 @@
   "homepage": "https://github.com/jedireza/mongo-models",
   "devDependencies": {
     "body-parser": "1.x.x",
-    "code": "5.x.x",
+    "@hapi/code": "5.3.x",
     "express": "4.x.x",
     "hapitoc": "1.x.x",
-    "lab": "15.x.x",
-    "mongodb": "3.x.x"
+    "@hapi/lab": "19.x.x",
+    "mongodb": "3.2.x"
```

Comment:
You can leave this at `3.x.x` since it will bring in the latest minor version during installation.

Reply:
I'm not a sure on this one. Between 3.0.0 and 3.1.0 they introduced a breaking change (the update atomic operator). My intent was to limit the version to 3.2.x to avoid this type of issue in a future update to mongodb.

Reply:
If we don't make the atomic operator assumption, we can keep this as is.

Reply:
Ok, sticking with 3.x.x